tests/regression_tests/filter_energyfun/test.py

```diff
@@ -28,18 +28,48 @@ def model():
     # Make an EnergyFunctionFilter directly from the x and y lists.
     filt1 = openmc.EnergyFunctionFilter(x, y)
 
+    assert filt1.interpolation == 'linear-linear'
+
+    with pytest.raises(ValueError):
+        filt1.interpolation = '5th order polynomial'
```

Comment:
Huge missed opportunity to throw your favorite unicode characters in here :wink:

Reply:
*frantically google searches for frisbee emoji*

Reply:
Haha, I already threw in a 🥏 (pun intended) in your honor in a different unit test, so now we have two of them :smile: